openedx/core/lib/request_utils.py

```diff
@@ -120,181 +120,168 @@ def __call__(self, request):
+# .. setting_name: IGNORED_ERRORS
 # .. setting_default: None
-# .. setting_description: Used to configure logging and monitoring for expected errors.
+# .. setting_description: Used to configure logging and monitoring for ignored errors.
 #     This setting is configured of a list of dicts. See setting and toggle annotations for
-#     ``EXPECTED_ERRORS[N]['XXX']`` for details of each item in the dict.
-#     If this setting is a non-empty list, all uncaught errors processed will get a ``checked_error_expected_from``
-#     attribute, whether they are expected or not. Those errors that are processed and match a 'MODULE_AND_CLASS'
-#     (documented elsewhere), will get an ``error_expected`` custom attribute. Unexpected errors would be errors with
-#     ``error_expected IS NULL``. For additional diagnostic information for ignored errors, see the
-#     EXPECTED_ERRORS[N]['IS_IGNORED'] annotation.
+#     ``IGNORED_ERRORS[N]['XXX']`` for details of each item in the dict.
+#     If this setting is a non-empty list, all uncaught errors processed will get a ``checked_error_ignored_from``
+#     attribute, whether they are ignored or not. See IGNORED_ERRORS[N]['MODULE_AND_CLASS'] annotation
+#     for details of monitoring added if the error is to be ignored.
 # .. setting_warning: We use Django Middleware and a DRF custom error handler to find uncaught errors. Some errors may
-#     slip through the cracks, like ValidationError. Any error where ``checked_error_expected_from IS NULL`` is
+#     slip through the cracks, like ValidationError. Any error where ``checked_error_ignored_from IS NULL`` is
 #     an error that was not processed.
 
-# .. setting_name: EXPECTED_ERRORS[N]['MODULE_AND_CLASS']
+# .. setting_name: IGNORED_ERRORS[N]['MODULE_AND_CLASS']
 # .. setting_default: None
-# .. setting_description: Required error module and class name that is expected. For example,
-#     ``rest_framework.exceptions.PermissionDenied``.
-
-# .. toggle_name: EXPECTED_ERRORS[N]['IS_IGNORED']
-# .. toggle_implementation: DjangoSetting
-# .. toggle_default: True
-# .. toggle_description: Set this to False if the errors are not ignored by monitoring, but only expected, like
-#      for temporary problems that may take some time to fix. If True, adds the custom attributes
-#      ``error_ignored_class`` and ``error_ignored_message`` to help diagnose issues with ignored errors, since
-#      this data is not otherwise available. For example of ignoring errors in New Relic, see:
+# .. setting_description: Required error module and class name that is ignored. For example,
+#     ``rest_framework.exceptions.PermissionDenied``. If the current error matches the module and class
+#      defined here, the middleware will add the custom attributes ``error_ignored_class`` and ``error_ignored_message``
+#.     to help diagnose issues with ignored errors, since this data is not otherwise available.
+#      For example of ignoring errors in New Relic, see:
```

Comment:
```suggestion
#      For an example of ignoring errors in New Relic, see:
```

Reply:
When I push code those changes come currently changes are in a local system. Thanks.